iso22133.i

```diff
@@ -4,33 +4,42 @@
 %{
 #include "iso22133.h"
 #include "positioning.h"
+#include "defines.h"
+#include "dreq.h"
+#include "dres.h"
+#include "footer.h"
+#include "grem.h"
+#include "header.h"
+#include "iohelpers.h"
+#include "monr.h"
+#include "osem.h"
+#include "ostm.h"
+#include "timeconversions.h"
+// #include "traj.h"
 #define SWIG_PYTHON_STRICT_BYTE_CHAR
 %}
 
-%apply int *OUTPUT {ObjectCommandType* command};
-%inline %{
-	extern ssize_t decodeOSTMMessage(const char* ostmDataBuffer, const size_t bufferLength, ObjectCommandType* command, const char debug);
-%}
-
-%apply int *OUTPUT {uint32_t *senderID}
-%inline %{
-extern ssize_t decodeOSEMMessage(ObjectSettingsType *objectSettingsData, const char * osemDataBuffer, const size_t bufferLength, uint32_t *senderID, const char debug);
-%}
 
 
 #%javaconst(1);
 
 typedef double double_t;
 typedef long int ssize_t;
 
-struct timeval {
```

Comment:
Why was this removed?

Reply:
it was an superfluous define, which caused problems when building SWIG. See positioning.i